core-feature-pack/common/src/main/resources/content/bin/installation-manager.bat

```diff
@@ -11,14 +11,21 @@ if "%INST_MGR_SCRIPT_DEBUG%"=="true" (
 
 set INSTALLATION_HOME=%~1
 set INST_MGR_LOG_PROPERTIES=%~2
+set INST_MGR_LOG_FILE=%~3
+
+rem Prepare variables for log traces
+set "TIMESTAMP=%date% %time%"
```

Comment:
I think this sets the `TIMESTAMP` to the same value each time it is used. IIRC one way around that is by using the `DelayedExpansion`, however it's explicitly enabled here. I don't know if we can use a batch call here or not, but that might also be an option.

Reply:
Right, it was a mistake, fixed